spec

```diff
@@ -0,0 +1,12532 @@
+
+    include "build/src/boot/voicecontwrite20.o"
+    include "build/src/boot/crc.o"
+    include "build/src/libultra/os/getactivequeue.o"
+    include "build/src/boot/normalize.o"
+
+    include "build/asm/boot/setcompare.text.o"
+
+    include "build/asm/boot/getcompare.text.o"
+
+    include "build/src/libultra/io/dpgetstat.o"
+    include "build/src/libultra/io/dpsetstat.o"
+    include "build/src/boot/bcopy.o"
+    include "build/src/boot/resetglobalintmask.o"
+    include "build/src/boot/pfsdeletefile.o"
+    include "build/src/libultra/gu/ortho.o"
+
+    include "build/asm/boot/interrupt.text.o"
+
+    include "build/src/libultra/io/vi.o"
+    include "build/data/boot/vi.data.o"
+
+    include "build/src/boot/viswapcontext.o"
+    include "build/src/libultra/io/pigetcmdq.o"
+
+    include "build/src/boot/cosf.o"
+
+    include "build/src/boot/epiread.o"
+    include "build/src/libultra/io/visetspecial.o"
+
+    include "build/src/libultra/gu/coss.o"
+
+    include "build/src/libultra/os/settime.o"
+    include "build/src/boot/voicestopread.o"
+    include "build/src/libultra/gu/visetevent.o"
+    include "build/src/boot/pfsisplug.o"
+
+    include "build/src/libultra/voice/voicegetstatus.o"
+
+    include "build/src/boot/cartrominit.o"
+    include "build/data/boot/cartrominit.data.o"
+    include "build/data/boot/cartrominit.bss.o"
+
+    include "build/src/boot/guS2DInitBg.o"
+    include "build/data/boot/guS2DInitBg.data.o"
+
+    include "build/src/boot/pfsselectbank.o"
+    include "build/src/libultra/io/contsetch.o"
+    include "build/src/boot/setfpccsr.o"
+    include "build/src/boot/getfpccsr.o"
+    include "build/src/boot/pfsfilestate.o"
+    include "build/src/boot/pfsinitpak.o"
+    include "build/src/boot/pfschecker.o"
+    include "build/src/libultra/io/aigetlen.o"
+    include "build/src/boot/epiwrite.o"
+
+    include "build/asm/boot/maptlbrdb.text.o"
+
+    include "build/src/libultra/os/yieldthread.o"
+    pad_text
+    include "build/asm/boot/guTranslate.text.o"
+
+    include "build/asm/boot/getcause.text.o"
+
+    include "build/src/boot/contramwrite.o"
+    include "build/src/libultra/io/epirawwrite.o"
+    include "build/src/libultra/os/settimer.o"
+
+    include "build/data/boot/__libm_qnan_f.rodata.o"
+
+    include "build/src/boot/xldtob.o"
+    include "build/data/boot/xldtob.data.o"
+
+    include "build/src/libultra/libc/ldiv.o"
+
+    include "build/src/boot/xlitob.o"
+    include "build/src/libultra/io/sirawwrite.o"
+    include "build/src/libultra/io/spgetstat.o"
+    include "build/src/libultra/io/spsetstat.o"
+
+    pad_text
+    include "build/src/libultra/io/vimgr.o"
+    include "build/data/boot/vimgr.data.o"
+    include "build/data/boot/vimgr.bss.o"
+
+    include "build/src/libultra/io/vigetcurrcontext.o"
+    include "build/src/boot/writebackdcacheall.o"
+    include "build/src/libultra/os/getcurrfaultthread.o"
+    include "build/src/boot/voicemaskdictionary.o"
+    pad_text
```

Comment:
Should these pads have newlines like the others?

Reply:
Oh, those newlines are my fault. I added them when trying to add the existing and boostrapped files to the spec. Those can go away if needed.